pom.xml

```diff
@@ -183,6 +183,10 @@
                     <groupId>org.scala-lang</groupId>
                     <artifactId>scala-library</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>com.twitter</groupId>
```

Comment:
It's an exclusion, we don't use this artifact and it's failing in Eclipse compilation.